lib/bigquery/dataset.js

```diff
@@ -28,13 +28,13 @@ var nodeutil = require('util');
  * @type {module:common/service-object}
  * @private
  */
-var ServiceObject = require('../common/service-object.js');
+var ServiceObject = require('@google-cloud/common/service-object.js');
```

Comment:
Do you think `common` should be more like a module and define access to these files in code, i.e.:

```js
var ServiceObject = require('@google-cloud/common').ServiceObject;
```